package.json

```diff
@@ -90,10 +90,10 @@
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
     "test": "npm run test:unit && npm run test:browser",
     "test:browser": "playwright test",
-    "test:unit": "vitest run",
-    "test:watch": "vitest watch",
-    "test:coverage": "vitest run --coverage",
-    "test:dev": "vitest watch --coverage",
+    "test:unit": "TZ=UTC vitest run",
```

Comment:
I bet we can solve this in the action running environment. But not going to mess with that now. This is fine.